tests/e2e/utils/utils.py

```diff
@@ -249,10 +249,10 @@ def uninstall_helm(chart_name, namespace=None):
     """
     if namespace:
         uninstall_retcode = subprocess.call(
-            f"helm uninstall {chart_name} -n {namespace} --wait".split()
+            f"helm uninstall {chart_name} -n {namespace}".split()
```

Comment:
impact of removing the --wait?

Reply:
Wait was introduced in #800, the intention was to introduce a synchronous wait to check if all resources were deleted prior to crd deletion. But there are problems with knative-serving being uninstalled, possibly timeout issues. I suspect that there may be some race condition.